src/Symfony/Component/Console/Command/SignalableCommand.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Console\Command;
+
+use Symfony\Component\Console\Exception\InvalidArgumentException;
+use Symfony\Component\Console\Exception\LogicException;
+
+/**
+ * @internal
+ */
+class SignalableCommand extends Command implements SignalableCommandInterface
```

Comment:
instead of adding a new class, would it make sense to implement SignalableCommandInterface on InvokableCommand and have it return an empty list by default?

Reply:
@nicolas-grekas Done, please let me know what you think of the new approach.